graylog2-web-interface/src/views/components/widgets/MessageList.jsx

```diff
@@ -95,7 +95,7 @@ class MessageList extends React.Component<Props, State> {
   _resetPagination = () => {
```

Comment:
The `_resetPagination` is getting executed on `SearchActions.execute.completed` Is there a way to trigger `SearchActions.execute.completed.listen(this._resetPagination);` inside the tests?

Reply:
Not that I know of..